src/components/pages/InstructorHome/InstructorSidebar.js

```diff
@@ -43,6 +43,10 @@ function InstructorSidebar() {
           <a href="/instructor">Dashboard</a>
         </Menu.Item>
 
+        <Menu.Item key="3" icon={<DesktopOutlined />}>
```

Comment:
Assign a unique key for this menu.item, there are currently two with the key="3".

Reply:
will do, thank you